tests/test_test_tools/test_test_cases.py

```diff
@@ -318,14 +318,6 @@ def test_diff_different_overrides(self):
         assert diff
         assert diff[0] == DEFAULT_AEA_CONFIG_FILE
 
-    def test_terminate_subprocesses(self):
```

Comment:
Why removed?

Reply:
because removed `_terminate_subprocesses`. Will put back for the time being